test_crates/enum_variant_missing/old/src/lib.rs

```diff
@@ -1,10 +1,21 @@
-pub enum VariantWillBeRemoved {
+pub enum PlainVariantWillBeRemoved {
     Foo,
-
     /// Testing: <https://doc.rust-lang.org/cargo/reference/semver.html#item-remove>
     Bar,
 }
 
+pub enum TupleVariantWillBeRemoved {
+    Foo(usize),
+    /// Testing: <https://doc.rust-lang.org/cargo/reference/semver.html#item-remove>
+    Bar(usize),
+}
+
+pub enum StructVariantWillBeRemoved {
+    Foo { bar: usize },
+    /// Testing: <https://doc.rust-lang.org/cargo/reference/semver.html#item-remove>
+    Bar{ bar: usize },
```

Comment:
Please make sure to `cargo fmt` test crate code.
```suggestion
    Bar { bar: usize },
```